src/components/header/index.module.scss

```diff
@@ -133,4 +82,50 @@
     border-right: 6px solid transparent;
     margin: 15px;
   }
-} 
\ No newline at end of file
+} 
+
+.imgContainer {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  width: 35px;
+  border-radius: 50%;
+  background-color: $white;
+}
+
+.selectContainer {
+  width: 270px;
+}
+
+select {
+  position: relative;
+  display: inline-block;
+  border: none;
+  margin-left: 10px;
```

Comment:
let's also add an `outline: none;` to this `select` css. Currently there's an outline when switching teams:

<img width="361" alt="Screen Shot 2020-05-01 at 12 51 44 AM" src="https://user-images.githubusercontent.com/623755/80782913-f48b5e00-8b45-11ea-9b31-7638e7c5211f.png">